resources/assets/uad_lists.json

```
@@ -34948,5 +34948,525 @@
+  },
+  "com.huawei.imedia.dolby": {
+    "list": "Oem",
+    "description": "Dolby Atmos\nOptimize sound automatically for exceptional audio quality.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.huawei.dtmfanalyzer": {
+    "list": "Oem",
+    "description": "Dtmf Analyzer\nHas something to media player to ride mode for China.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.easygo": {
+    "list": "Oem",
+    "description": "HwEasyGo\nLooks like a backdoor for WeChat app?\nEasyGoServer, error code things found, also EasyGoCallBack.\nThese app is probably backdoor or for developers.\nThis app has no permissions and no activities.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.hwesimservice": {
+    "list": "Oem",
+    "description": "HwESIMService\nIf you use eSIM do not remove it.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.huawei.manufacture.wificonnect": {
+    "list": "Oem",
+    "description": "Wifi Connect\nNeeded for WiFi connection?",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.huawei.fans": {
+    "list": "Oem",
+    "description": "Huawei Club is the official forum for Huawei users.\nIt provides the latest news and information, FAQs, and tutorials about Huawei and Honor products.",
+    "dependencies": ["com.huawei.hwid"],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.imonitor": {
+    "list": "Oem",
+    "description": "imonitor\nHidden logs founded in app.\nDetects app launch anomaly and runs safe mode.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.internetaudioservice": {
+    "list": "Oem",
+    "description": "Smart headset control\nHas a lot logs and frameworks to mmitest.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.lives": {
+    "list": "Oem",
+    "description": "HiLives is jointly developed by Huawei and certified third-party service providers to provide lifestyle services for you.",
+    "dependencies": ["com.huawei.hwid"],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.featurelayer.sharedfeature.map": {
+    "list": "Oem",
+    "description": "Huawei Map Service\nRequire FeatureFramework.\nOnly uses Chinese AMap to get location.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  "com.huawei.notepad": {
+    "list": "Oem",
+    "description": "Notepad\nLooks so bloated with Hi Voice, Update using AppGallery, Location.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
```

```suggestion
    "removal": "Recommended"
```
If it's very bloated and doesn't break anything, then shouldn't it be `Recommended`?

---

wait, it has less privacy issues, but may be recommended